lib/components/Insights/InsightsContainer/index.tsx

```diff
@@ -21,7 +21,7 @@ export interface InsightsContainerProps {
     title: string
     description: string
   }
-  footer: {
+  footer?: {
```

Comment:
I guess it's fine if footer is optional? In any case this should change according to what @dani-moreno and Desirée end up with.